python/deltalake/writer.py

```diff
@@ -44,6 +44,9 @@ def write_deltalake(
     partition_by: Optional[List[str]] = None,
     filesystem: Optional[pa_fs.FileSystem] = None,
     mode: Literal["error", "append", "overwrite", "ignore"] = "error",
+    name: Optional[str] = None,
+    description: Optional[str] = None,
+    configuration: Optional[Mapping[str, Optional[str]]] = None,
```

Comment:
Could you add entries to the docstring below for each of these parameters?

Reply:
Done 👍